src/utils/ProtocolMetrics.ts

```diff
@@ -526,7 +526,7 @@ function getRunway(totalSupply: BigDecimal, rfv: BigDecimal, rebase: BigDecimal,
         let runway50k_num = (Math.log(treasury_runway) / Math.log(1+0.00569158))/3;
         let runway70k_num = (Math.log(treasury_runway) / Math.log(1+0.00600065))/3;
         let runway100k_num = (Math.log(treasury_runway) / Math.log(1+0.00632839))/3;
-        let nextEpochRebase_number = Number.parseFloat(rebase.toString())/100
+        let nextEpochRebase_number = Number.parseFloat(rebase.toString())
```

Comment:
i think this is the only other place where it was used.. and we had to back out the logic